includes/discovery/services.inc.php

```diff
@@ -1,35 +1,54 @@
 <?php
+/**
+ * services.inc.php
+ *
+ * Creates the correct handler for the trap and then sends it the trap.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Autodiscovers TCP services on devices in the 'server' device type.
+ * To use set in config.php:
+ * [discover_services] = 'true';
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2019 KanREN, Inc.
+ * @author     Heath Barnhart <hbarnhart@kanren.net>
+ */
 
 use LibreNMS\Config;
 
 if (Config::get('discover_services')) {
```

Comment:
This seems silly, shouldn't we just use the regular module enable/disable settings?

Reply:
That's what was there, I was just taking what was existing and improving on it. Mostly been working on background stuff like the snmphandlers. I'll look into the how to interact with the GUI.

Makes sense though.

Reply:
Or to look at it another way, I was fixated on the making /etc/services work, didn't even think about the the GUI toggle.